x-pack/plugins/maps/public/connected_components/mb_map/mb_map.tsx

```diff
@@ -37,12 +33,10 @@ import {
   RawValue,
   ZOOM_PRECISION,
 } from '../../../common/constants';
-import { getGlyphUrl, isRetina } from '../../util';
+import { getGlyphUrl } from '../../util';
 import { syncLayerOrder } from './sort_layers';
 
 import {
-  addSpriteSheetToMapFromImageData,
-  loadSpriteSheetImageData,
   removeOrphanedSourcesAndLayers,
 } from './utils';
```

Comment:
With this change, `addSpriteSheetToMapFromImageData` and `loadSpriteSheetImageData` are now only used by the `VectorTileLayer` module. Not sure if we want to continue to export these functions from `utils` or if we should move them to `vector_tile_layer.tsx`?

Reply:
I would still keep then in a separate file, How about just moving them into vector_tile_layer folder?